owncloud/core
# Make sender display name in mail notifications configurable

## Description
Make sender display name in mail notifications configurable.

## Motivation and Context
In some cases mail notifications related to sharing activities are blocked by mail filters as they are flagged as email impersonation. In such cases it may be desirable for an oC admin to have a config option for removing the sender display name.

## How Has This Been Tested?

- Set `remove_sender_display_name => true` in config.php. The `From` address in the mail notifications is displayed as follow:

<img width="615" alt="image" src="https://user-images.githubusercontent.com/12248713/223242596-38033a0c-92b0-476f-a5ee-1f9baab1de61.png">

- Set `remove_sender_display_name => false` in config.php / do not set it at all. The `From` address in the mail notifications is displayed as per current behaviour:

<img width="621" alt="image" src="https://user-images.githubusercontent.com/12248713/223242900-32d18f3e-0193-4228-b799-17660a873958.png">

## Types of changes

## Checklist:
- [ ] Documentation ticket raised
- [ ] Changelog item
